openshift/setup-teardown/params/CUSTOM_SETTINGS/OCP4_PR/api/dev/api-3-nodejs-dc.params

```diff
@@ -17,3 +17,14 @@ NODEJS_CPU_LIMIT=250m
 NODEJS_MEMORY_REQUEST=250Mi
 NODEJS_MEMORY_LIMIT=500Mi
 ENABLE_VIRUS_SCANNING=true
+GETOK_ENDPOINT=https://dev.oidc.gov.bc.ca/auth/realms/jbd6rnxw/protocol/openid-connect/token
+CHES_ENDPOINT=https://ches-dev.pathfinder.gov.bc.ca/
+SSO_ISSUER=https://dev.oidc.gov.bc.ca/auth/realms/eagle
+SSO_JWKSURI=https://dev.oidc.gov.bc.ca/auth/realms/eagle/protocol/openid-connect/certs
+KEYCLOAK_CLIENT_ID=eagle-admin-pr-console
+KEYCLOAK_URL=https://dev.oidc.gov.bc.ca/auth
+KEYCLOAK_REALM=eagle
+KEYCLOAK_ENABLED=true
+API_LOCATION=https://eagle-dev.pathfinder.gov.bc.ca
```

Comment:
I think this should be `https://eagle-pr-placeholder-dev.apps.silver.devops.gov.bc.ca`?

Reply:
whoops yep!